src/boot/store.js

```diff
@@ -343,7 +345,8 @@ const reduxPersistConfig: Config = {
 
   // Store data through our own wrapper for AsyncStorage, in particular
   // to get compression.
-  // $FlowFixMe: https://github.com/rt2zz/redux-persist/issues/823
+  // https://github.com/rt2zz/redux-persist/issues/823
+  // $FlowFixMe[incompatible-variance]
   storage: ZulipAsyncStorage,
```

Comment:
This is another nice illustration of how it's helpful to keep the text explaining the fixme explicitly attached to the fixme. As is, it looks like this issue link is probably there to add more context on the "Store data through … compression" comment, rather than to explain the fixme.